src/job_metadata.c

```diff
@@ -275,6 +281,43 @@ NextJobId(void)
 	return jobId;
 }
 
+int64
+NextRunId(void)
```

Comment:
There are some style things like having comments on all functions, but I think it's easiest to merge and adjust them later.